src/app/patient-dashboard/patient-identifier/patient-identifier.component.spec.ts

```diff
@@ -0,0 +1,11 @@
+/* tslint:disable:no-unused-variable */
+
```

Comment:
Write some useful test to check whether your component is rendered properly see the visit component